deploy/charts/rook-ceph/templates/configmap.yaml

```
@@ -77,6 +77,9 @@ data:
   ROOK_CSI_RESIZER_IMAGE: {{ .Values.csi.resizer.image | quote }}
 {{- end }}
 {{- end }}
+{{- if .Values.csi.imagePullPolicy }}
+  ROOK_CSI_IMAGE_PULL_POLICY: {{ .Values.csi.imagePullPolicy | quote }}
+{{- end }}
```

```suggestion
{{- if .Values.csi.imagePullPolicy }}
  ROOK_CSI_IMAGE_PULL_POLICY: {{ .Values.csi.imagePullPolicy | quote }}
{{- end }}
{{- end }}
```

This should go inside if check for empty `.Value.csi`

---

I think it's alreayd inside, as end for .Values.csi is at line 221